### ISSUE TITLE
Consolidate list of department names across the FSDH

### ISSUE THREAD

**Issue by yjmrobert** [ssc-sp/datahub-portal#646]
### Bug type

DataHub Portal, Other

### What happened?

A user has reported that there are differing names for some of the departments (Natural Resources Canada vs Department of Natural Resources).

There are a few instances of this list being used, off the top of my head
- Registration page
- Metadata page
- Support Request Form

### Expected behavior

They should all use the same list in both french and english

### Reproduction steps

1.
2.
3.
...

### What browsers are you seeing the problem on?

Chrome

### On what operating system are you experiencing the issue?

Windows

### Code of Conduct

- [X] I agree that none of the information provided is confidential.

**Comment by iroelperez** [comment]
I don't know where the **Registration page** or the **Support Request Form** got the list from, in the case of the **Metadata page** I used the Open Data dataset provided by Lynne. Maybe we can try to use that one everywhere.

**Comment by yjmrobert** [comment]
```
https://open.canada.ca/data/en/api/3/action/datastore_search?resource_id=f0ca63e0-c15e-45b5-9656-77abe1564b1c
```

**Comment by Lynnehunks** [comment]
I submitted the question below to the Open Government Portal. We will make decision on list to use once we hear back from them. (Note - Irum advised to use the first list)

Subject
Department List -- what is the official list

Comments and Feedback
We are trying to use the correct list of departments for our application but have found 2 sources. What is the correct source to use? 
1. Official titles of Government of Canada departments and agencies (https://open.canada.ca/data/dataset/83320390-7715-43bc-a281-2049bf5d4232/resource/f0ca63e0-c15e-45b5-9656-77abe1564b1c/download/fip-registry-of-applied-titles-registre-des-titres-dusage-du-pfim-p1-20220427.csv) 

OR

2. Open Government Portal Department List

https://open.canada.ca/data/dataset/933c7f9d-deb0-4367-940d-06c38f494153/resource/04cbec5c-5a3d-4d34-927d-e41c9e6e3736/download/orgs.csv

**Comment by Lynnehunks** [comment]
We have confirmed the correct source and dataset for the list of Applied titles that is used on the Workspace Metadata page in the portal. I have updated the Metadata profile document:   FSDH Workspace Metadata - FINAL v3 revised.docx.
Also including the csv link here: -- Applied title to be used, and Legal title where no Applied title exists -- except for second Treasury Board entry and GC  (https://open.canada.ca/data/dataset/83320390-7715-43bc-a281-2049bf5d4232/resource/f0ca63e0-c15e-45b5-9656-77abe1564b1c/download/fip-registry-of-applied-titles-registre-des-titres-dusage-du-pfim-p1-20220427.csv)
-- Iroel will make the changes in the database on Tuesday.

Not sure what the steps are to finish the ‘consolidation’ ...
